pyvrp/cpp/educate/Node.h

```diff
@@ -16,9 +17,9 @@ class Node
     Route *route;     // Pointer towards the associated route
 
     // TODO can these data fields be moved to Route?
-    int cumulatedLoad;              // Load from depot to client (inclusive)
-    int cumulatedDistance;          // Distance from depot to client (inclusive)
-    int cumulatedReversalDistance;  // Distance if (0 .. client) is reversed
+    int cumulatedLoad;                // Load from depot to client (incl)
+    distance_type cumulatedDistance;  // Distance from depot to client (incl)
+    distance_type cumulatedReversalDistance;  // Reverse distance of (0..client)
```

Comment:
Does this specify the `delta` of the distance between the forward and backward direction, or the actual distance of the reversed segment? From my head it was the first, but the comment now reads as if it's the second. Also, if this is inclusive, maybe use `[0, ..., client]` to indicate that?

Reply:
It's the $\Delta$, and I've updated the comments/names to reflect this.